_scss/_uswds-theme-custom-styles.scss

```diff
@@ -21,3 +21,19 @@ i.e.
 */
 
 @import "components/glossary";
+
```

Comment:
Definitely not a requirement, but if you'd like to create a new `footer` component file in `components/` and move your custom css to it and `@import "components/footer";` here, it may help keep things tidy as we continue to customize the different components.